redash/query_runner/mssql.py

```diff
@@ -67,6 +67,16 @@ def name(cls):
     def type(cls):
         return "mssql"
 
+    # Override the base class's host property to alias 'server'. 
```

Comment:
I realize this query runner is effectively deprecated. I can amend the commit to remove this change if it's best to not change this code now.